File: gaming/src/main/java/com/oruga/gaming/dto/JuegoDto.java
package com.oruga.gaming.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class JuegoDto {
    private Long id;
    private String nombre;
    private String descripcion;
    private String genero;
    private Double precio;
    private String precioFormateado;
    private Integer stock;
    private String imagenUrl;
    private String plataformas;
    private String desarrollador;
    private String fechaLanzamiento;
    private Double calificacion;
    private Boolean activo;
}

File: MyApplication3/app/src/main/java/com/example/myapplication2/data/remote/dtos/JuegoDto.kt
package com.example.myapplication.data.remote.dtos

data class JuegoDto(
    val id: Long,
    val nombre: String,
    val descripcion: String,
    val genero: String,
    val precio: Double,
    val precioFormateado: String,
    val stock: Int,
    val imagenUrl: String?,
    val plataformas: String?,
    val desarrollador: String?,
    val fechaLanzamiento: String?,
    val calificacion: Double?,
    val activo: Boolean,

)